packages/orchestration/src/joiner/graphql-ast.ts

```diff
@@ -75,6 +82,33 @@ class GraphQLParser {
     })
   }
 
+  private parseDirectives(directives: readonly DirectiveNode[]): Directive[] {
```

Comment:
comment: I like the idea of directives, but not sure we should put too much effort into this one right now. Would like us to dedicate some time to thinking about the DX. 

This is not consumed by anything yet, so I guess we can keep it in for now

Reply:
That won't advance for now, but I believe it will be useful in the near future.
I think we can use it for things like:

```graphql
query {
  products {
    title
    variants @count {
      id
    }
  }
}
```
Where we just want to count the rows, and not list them.

And the RemoteQuery can be extended to resolve custom queries. That can give an extra layer of flexibility to proccess things on the server/module side before responding the data.